ghwt.py

```diff
@@ -40,7 +40,7 @@ def list_projects():
     return 0
 
 def unpack(sproj, branch, outdir):
-    subprocess.check_call(['git', 'clone', '-b', branch, 'https://github.com/mesonbuild/%s.git' % sproj, outdir])
+    subprocess.check_call(['git', 'clone', '-b', branch, 'https://github.com/mesonbuild/{0}.git'.format(sproj, outdir)])
```

Comment:
`outdir` is not an argument to % in this case, it's the final element of the array.

Reply:
Problem fixed.  Just moved subprojects variable out of format method.